danielforkner.Calculator/danielforkner.Calculator/Program.cs

```diff
@@ -0,0 +1,216 @@
+﻿using CalculatorLibrary;
+
+namespace CalculatorProgram
+{
+
+    class Program
+    {
+
+        public static void ViewOperations(Calculator calculator)
+        {
+            Console.WriteLine("\n----- Viewing Past Operations -----\n");
+            bool viewingOperations = true;
+            int idx = 0;
+            while (viewingOperations)
+                {
+                    if (calculator.Operations.Count == 0)
+                    {
+                        Console.WriteLine("You don't have any past operations to view!");
+                        viewingOperations = false;
+                        continue;
+                    }
+                    // operation
```

Comment:
Maybe these comments came from Microsoft Docs, but usually refrain from commenting what can be explained by the code itself. Code should be self explanatory. The better you name your members, arguments, classes, the less comments you need.

Reply:
Good point, removed!